package.json

```diff
@@ -9,6 +9,7 @@
   "license": "ISC",
   "description": "",
   "dependencies": {
+    "@adobe/rum-distiller": "github:adobe/rum-distiller",
```

Comment:
Why install from GitHub and not from npm? https://www.npmjs.com/package/@adobe/rum-distiller
```
@adobe/rum-distiller
```